src/cholesky.jl

```diff
@@ -668,14 +668,14 @@ function _choleskyUfactor(Cfactors, Cuplo)
     if Cuplo === 'U'
         return UpperTriangular(Cfactors)
     else
-        return copy(LowerTriangular(Cfactors)')
+        return LowerTriangular(Cfactors)'
```

Comment:
I was slightly confused when reading this the first time, IMO from
```suggestion
        return UpperTriangular(Cfactors')
```
it would have been more obvious that in both branches an `UpperTriangular` is returned.

Reply:
Yes, we can do that as well.

Reply:
We should get this one in too.

Reply:
Won't this create a copy?

Reply:
No, the `UpperTriangular` just wraps the lazy adjoint.